app/templates/index.html

```diff
@@ -0,0 +1,10 @@
+{% extends  "base.html" %}
+
+
+{% block content %}
+    <h1>DRC Web-Hook app is coming soon.</h1>
+    <br>
+    <h4>What is DRC Web-Hook app?</h4>
+    <p>It is an application that would receive data from farmers and responds based on the information by <b>Chris</b> that would make the life of farmers easy but more productive.</p>
+    <p>Please see also <a href="https://www.youtube.com/watch?v=qkBsuUHCFWo" target='blank'>Mulungwishi Fertilizer Project</a></p>

```

Comment:
`blank` is not a valid value for the `target` attribute. [Reference](http://www.w3schools.com/tags/att_a_target.asp)

Also, use "double quotes" in HTML.